CONTRIBUTING.md

```diff
@@ -126,3 +126,12 @@ yarn start
 
 Now visit http://localhost:3000 and you should see a local version of the
 website. New changes should trigger a reload in the browser.
+
+## Back publishing
+
+To release a new version of the SemanticDB compiler plugin, you need to back publish it for a specific Scala version. To do this push a tag to the scalameta repository in a form of:
+
+  `v4.13.43@2.13.17@semanticdbScalacCore/publishSigned@semanticdbScalacPlugin/publishSigned`
```

Comment:
what causes this to be processed as described? also, it looks like this includes some sbt targets, but lists core in addition to scalacPlugin (and not metac etc. that were in the code above), why is that?

Reply:
It's here https://github.com/sbt/sbt-ci-release?tab=readme-ov-file#back-publishing-support 

I can add all the sbt targets, I haven't because they are not needed for Metals in reality.

Reply:
> It's here https://github.com/sbt/sbt-ci-release?tab=readme-ov-file#back-publishing-support

- in this document, it seems that only one command is allowed, am i misreading this?
- does `semanticdbScalacCore/publishSigned@semanticdbScalacPlugin/publishSigned` really work, in practice?
- and why isn't `semanticdbScalacPlugin/publishSigned` sufficient for publishing just the plugin?
- if we need to run multiple -- or lengthy -- commands, would it make sense to add an alias to `build.sbt`, to avoid "fat fingers"?

Reply:
I think it works by switching @ to ;, but it might be a good idea to just create a command.

Reply:
Changed to using a single command, should be much nicer to work with now.